client/my-sites/domains/domain-management/list/bulk-site-domains.tsx

```diff
@@ -129,20 +127,20 @@ export default function BulkSiteDomains( props: BulkSiteDomainsProps ) {
 							setChangeSiteAddressSourceDomain( domain );
 						}
 					} }
+					footer={
```

Comment:
The reason why I'm adding this `footer` element is because the stickiness of the toolbar applies only whenever the `.domains-table` element is in the viewport. Once it's off (meaning the table is not showing anymore), the stickiness effect disappears and the toolbar is gone:

https://github.com/Automattic/wp-calypso/assets/26530524/8e6a6ca5-d1e7-4b6c-b6ac-b0341fc0cb6d

I'm open for ideas but I'll merge this PR anyway to have the CTAs back. One annoying thing is that we'll need to add every element to this page inside this `footer` fragment, which is not ideal. But I don't think we'll add many elements: the footer is the last one in the old table.